spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -114,6 +114,33 @@ public class Trader {
 Spring Data Spanner will ignore any property annotated with `@NotMapped`. These properties will
 not be written to or read from Spanner.
 
+==== Constructors
+
+Simple constructors are supported on POJOs. The constructor arguments can be a subset of the persistent properties or all of them. Constructor arguments that do not map to a persistent property are not supported.
```

Comment:
You can leave out "or all of them". subsets also include the set itself

Reply:
done